packages/features/ee/workflows/lib/reminders/smsReminderManager.ts

```diff
@@ -139,180 +139,140 @@ export const scheduleSMSReminder = async (args: ScheduleTextReminderArgs) => {
   }
   const isNumberVerified = await getIsNumberVerified();
 
-  let attendeeToBeUsedInSMS: AttendeeInBookingInfo | null = null;
-  if (action === WorkflowActions.SMS_ATTENDEE) {
-    const attendeeWithReminderPhoneAsSMSReminderNumber =
-      reminderPhone && evt.attendees.find((attendee) => attendee.email === evt.responses?.email?.value);
-    attendeeToBeUsedInSMS = attendeeWithReminderPhoneAsSMSReminderNumber
-      ? attendeeWithReminderPhoneAsSMSReminderNumber
-      : evt.attendees[0];
-  } else {
-    attendeeToBeUsedInSMS = evt.attendees[0];
-  }
-
   if (triggerEvent === WorkflowTriggerEvents.BEFORE_EVENT) {
     scheduledDate = timeSpan.time && timeUnit ? dayjs(startTime).subtract(timeSpan.time, timeUnit) : null;
   } else if (triggerEvent === WorkflowTriggerEvents.AFTER_EVENT) {
     scheduledDate = timeSpan.time && timeUnit ? dayjs(endTime).add(timeSpan.time, timeUnit) : null;
   }
 
-  const name = action === WorkflowActions.SMS_ATTENDEE ? attendeeToBeUsedInSMS.name : "";
-  const attendeeName =
-    action === WorkflowActions.SMS_ATTENDEE ? evt.organizer.name : attendeeToBeUsedInSMS.name;
-  const timeZone =
-    action === WorkflowActions.SMS_ATTENDEE ? attendeeToBeUsedInSMS.timeZone : evt.organizer.timeZone;
-
-  const locale =
-    action === WorkflowActions.SMS_ATTENDEE
-      ? attendeeToBeUsedInSMS.language?.locale
-      : evt.organizer.language.locale;
+  if (reminderPhone && isNumberVerified) {
+    const useTwilio = shouldUseTwilio(triggerEvent, scheduledDate);
+    if (useTwilio) {
```

Comment:
No logic changed, the only thing that changed it hat we now only create th sms message (with `bulkShortenLinks`) when `useTwilio` is true